propagators/opentelemetry-propagator-aws-xray/README.md

```diff
@@ -0,0 +1,77 @@
+# OpenTelemetry Propagator AWS X-Ray
+
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+The OpenTelemetry Propagator for AWS X-Ray provides HTTP header propagation for systems that are using AWS `X-Amzn-Trace-Id` format.
+This propagator translates the OpenTelemetry SpanContext into the equivalent AWS header format, for use with the OpenTelemetry JS SDK.
```

Comment:
I think it should be mentioned that `TraceState` is not propagated

Reply:
Just for my interest, not directly effecting the PR.
If I understand this correct users have to use this propagator if they use AWS services because W3C tracecontext is not propagate there. But tracestate is lost. Are there any plans to preserve tracestate?

Reply:
Good question - we don't currently have a spec for modeling tracestate with the xray propagator. We should probably be able to do so at some point but haven't prioritized it - when we have a spec we'd apply it to all languages. Before tracestate, we're focusing on baggage which I'll add in a followup.